src/config/functions.ts

```diff
@@ -149,5 +149,12 @@ export function createProviderFunctions(
 				]);
 			},
 		},
+		unmanaged: {
```

Comment:
I guess this replaces offlineMode then? should we globally replace it rather than keeping both? (I don't think anyone is using the supervisorOfflineMode config.json flag and it's undocumented anyways, so I think we can do the breaking change without problems).